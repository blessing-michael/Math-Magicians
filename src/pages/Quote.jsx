import React from 'react';
import { DisplayQuote } from '../components';

const Quote = () => (
  <DisplayQuote />
);

export default Quote;
