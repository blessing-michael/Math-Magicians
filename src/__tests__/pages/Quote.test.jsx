import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../../pages/Quote';

describe('Quote', () => {
  test('Quote Page is rendered and matches the snapshot', () => {
    const { container } = render(<Quote />);
    expect(container).toMatchSnapshot();
  });
});
