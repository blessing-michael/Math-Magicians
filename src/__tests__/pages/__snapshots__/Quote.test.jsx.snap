// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Quote Page is rendered and matches the snapshot 1`] = `
<div>
  <div
    class="loader"
  >
    <img
      alt="loader"
      class="loader-img"
      src="loader.svg"
    />
    <p
      class="loader-tag"
    >
      We are getting you a quote
      <br />
      Please wait...
    </p>
  </div>
</div>
`;
